physics/biot_savart_spacepy.py

```diff
@@ -46,13 +46,17 @@
     Event = data[i, :]
 
     # read in the 3d magnetosphere
-    filename = time2filename(time, extension='.out')
```

Comment:
See how I handled looping over events in cutplane_demo2.py. We should use a similar approach.